chroot/run.go

```diff
@@ -894,7 +894,7 @@ func setCapabilities(spec *specs.Spec, keepCaps ...string) error {
 	capMap := map[capability.CapType][]string{
 		capability.BOUNDING:    spec.Process.Capabilities.Bounding,
 		capability.EFFECTIVE:   spec.Process.Capabilities.Effective,
-		capability.INHERITABLE: []string{},
+		capability.INHERITABLE: {},
```

Comment:
This was added here https://github.com/containers/buildah/pull/3853 , I am just running `gofmt` here so smoke test passes.

Reply:
@nalind @giuseppe Could you please review change in this line.

Reply:
An empty slice either way, so I don't see an issue with it.

Reply:
Strange that gofmt changed it though.